src/qualimap/qualimap_rnaseq/test.sh

```diff
@@ -0,0 +1,74 @@
+set -e
+
+test_dir="$meta_resources_dir/test_data"
+
+mkdir "run_qualimap_rnaseq_html"
+cd "run_qualimap_rnaseq_html"
+
+echo "> Running qualimap with html output report"
+
+"$meta_executable" \
+    --input $test_dir/a.bam \
+    --gtf $test_dir/annotation.gtf \
+    --output_report report.html \
+    --output_counts counts.txt \
+    --output output.txt
+
+echo ">> Checking output"
+[ ! -f "report.html" ] && echo "report.html does not exist" && exit 1
+[ ! -f "counts.txt" ] && echo "counts.txt does not exist" && exit 1
+[ ! -f "output.txt" ] && echo "output.txt does not exist" && exit 1
+
+echo ">> Checking if output is empty"
+[ ! -s "report.html" ] && echo "report.html is empty" && exit 1
+[ ! -s "counts.txt" ] && echo "counts.txt is empty" && exit 1
+[ ! -s "output.txt" ] && echo "output.txt is empty" && exit 1
```

Comment:
Can we check whether the outputs make sense?

Reply:
In the meantime, I added a bit more info to the contributing guidelines on what I mean by this: https://github.com/viash-hub/biobox/blob/main/CONTRIBUTING.md#step-12-create-test-script :relaxed: